routers/api/packages/maven/maven.go

```diff
@@ -215,7 +215,7 @@ func servePackageFile(ctx *context.Context, params parameters, serveContent bool
 		return
 	}
 
-	s, u, _, err := packages_service.GetPackageBlobStream(ctx, pf, pb)
+	s, u, _, err := packages_service.GetPackageBlobStream(ctx, pf, pb, nil)
```

Comment:
I'm not sure if other package managers also have content type issues, so I didn't specify a content type for them.